tests/unit/test_global_config_update.py

```diff
@@ -207,3 +208,31 @@ def test_config_validation_when_placeholder_is_present(tmp_path, caplog):
     expected_schema_version = "0.0.7"
     assert expected_schema_version == global_config.schema_version
     assert error_log in caplog.text
+
+
+def test_dump_enable_from_global_config_enable_present(tmp_path, caplog):
+    tmp_file_gc = tmp_path / "globalConfig.json"
+
+    helpers.copy_testdata_gc_to_tmp_file(
+        tmp_file_gc, "valid_config_input_with_enable_action.json"
+    )
+    global_config = global_config_lib.GlobalConfig(str(tmp_file_gc))
+    expected_schema_version = "0.0.9"
+
+    _dump_enable_from_global_config(global_config)
+
+    assert expected_schema_version == global_config.schema_version
+    assert "`enable` attribute found in input's page table action." in caplog.text
+
+
+def test_dump_enable_from_global_config_enable_absent(tmp_path, caplog):
+    tmp_file_gc = tmp_path / "globalConfig.yaml"
+
+    helpers.copy_testdata_gc_to_tmp_file(tmp_file_gc, "valid_config.yaml")
+    global_config = global_config_lib.GlobalConfig(str(tmp_file_gc))
+    expected_schema_version = "0.0.9"
+
+    _dump_enable_from_global_config(global_config)
+
+    assert expected_schema_version == global_config.schema_version
+    assert caplog.text == ""
```

Comment:
those tests are related with uccignore?

Reply:
The changes are related to that "enable" feature. However, I had to add this here because unit tests were failing only for Py3.10 as coverage was less than 81%, and was passing for the rest of the Py versions.